src/node_v8_platform-inl.h

```diff
@@ -99,6 +107,10 @@ struct V8Platform {
   }
 
   inline void Dispose() {
+    if(!initialize_)
```

Comment:
Disposing without initializing first sounds like a buggy use case to me. Why do we return instead of aborting here?

Reply:
https://github.com/nodejs/node/blob/bd6d6512f02d6fa10a0484577c270bb44f1b750b/src/env.cc#L958

Here is the part that calls DisposePlatform globally.

If created through CreatePlatform then v8_platform is not initialized.
So should be ignored instead of Aborting.

Reply:
In that case I think the state should be maintained in the caller instead of in the callee..

Reply:
DrainTasks now run after RunAtExit has run.
Is this order of execution important?
If order is not important, I think can Dispose with AtExit.